sunpy/net/tests/test_fido.py

```diff
@@ -2,6 +2,14 @@
 import copy
 import tempfile
 
+# Remove try/except in 1.0.
+# Python 2 doesnot have pathlib
+try:
+    import pathlib
+    HAS_PATHLIB = True
+except ImportError:
+    HAS_PATHLIB = False
```

Comment:
Would we not need this for the main file as well?

Reply:
Fixed!

Reply:
instead of doing this we should use `pytest.importorskip("pathlib")` in the test